test/cloud-slack-dev-e2e/page_helpers_test.go

```diff
@@ -0,0 +1,91 @@
+//go:build cloud_slack_dev_e2e
+
+package cloud_slack_dev_e2e
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/go-rod/rod"
+	"github.com/go-rod/rod/lib/proto"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	// Chromium is not supported by Slack web app for some reason
+	// Currently, we get:
+	//   This browser won’t be supported starting September 1st, 2024. Update your browser to keep using Slack. Learn more:
+	//   https://slack.com/intl/en-gb/help/articles/1500001836081-Slack-support-life-cycle-for-operating-systems-app-versions-and-browsers
```

Comment:
@pkosiec, looks like you could have similar problems, do you know if that's sth that we should be afraid of?

I saw that the latest Chrome version is 126.0, while Slack already "deprecated" it.

Reply:
Not sure, weird as the user agent seems to be the latest one 🤔  Let's keep it as it is for now 👍